cmd/webhook.go

```diff
@@ -220,9 +115,9 @@ func (o *webhookV2Options) validate(c *config.Config) error {
 	return nil
 }
 
-func (o *webhookV2Options) scaffolder(c *config.Config) (scaffold.Scaffolder, error) { //nolint:unparam
+func (o *webhookV2Options) scaffolder(c *config.Config) (scaffold.Scaffolder, error) { // nolint:unparam
```

Comment:
```suggestion
func (o *webhookV2Options) scaffolder(c *config.Config) (scaffold.Scaffolder, error) { //nolint:unparam
```

These spaces were recently removed: read https://github.com/kubernetes-sigs/kubebuilder/pull/1368#discussion_r379111906 for the justification

Reply:
Can be marked as solved.